packages/components/pie-modal/src/index.ts

```diff
@@ -59,14 +62,18 @@ export class PieModal extends RtlMixin(LitElement) {
     }
 
     disconnectedCallback () : void {
+        super.disconnectedCallback();
```

Comment:
The Lit docs have the `super` call at the start of the method so we probably should too
https://lit.dev/docs/components/lifecycle/#disconnectedcallback

Reply:
It might not make a difference but i think it kinda makes sense calling it last because as i understand it the disconnected super call cleans up the overall class. If we call it first there might be other calls inside here that dont get cleaned up? Saying that happy for you to change it but would be interesting to ask the LIT team about this. I could put a message in the room and ask.

Reply:
LIT team:

`It won't matter at all in that case. The LitElement disconnectedCallback just notifies AsyncDirectives and Reactive Controllers that they're disconnected
My advice based on code style is to put the super.disconnectedCallback(); at the top of the method, because that's where the super() call in a constructor must go, but that's mainly for rease of reading`